gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,6 +1,6 @@
-#Fri Sep 02 10:49:19 BST 2016
+#Tue Jan 28 19:31:47 CET 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.14.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip
```

Comment:
I think, you should run the following command:
`./gradlew wrapper --gradle-version=5.4.1 --distribution-type=all`, it will update:
- gradlew
- gradlew.bat
- ...

Reply:
Done. If I remember correctly, this was no longer updating anything around gradle version 4 and 5, but there were a lot of changes since v2.14.